.golangci.yml

```diff
@@ -19,7 +19,7 @@ linters-settings:
     local-prefixes: github.com/golangci/golangci-lint
 
   govet:
-    check-shadowing: true
+    shadow: true
```

Comment:
According to the latest docs this should be: 
https://golangci-lint.run/usage/linters/#govet

```yaml
shadow:
  strict: true
```